pkg/server/server.go

```diff
@@ -115,17 +225,28 @@ func tryApplyChanges(j jira.Client, patcher patch.Patcher, release Release, issu
 		})
 		return
 	}
+	createDynamicComment(j, issueRef, prs, release.Project, &cfg.Jira.Issue.Comments, tmplCtx)
```

Comment:
The `patcher.CloneAndPublishAll` code should only run when not setting `PRDeferredCreation`